exercises/two-bucket/example.js

```diff
@@ -1,89 +1,86 @@
 'use strict';
 
-function TwoBucket(x, y, z, starter) {
-  this.starter = starter;
-  this.x = x;
-  this.y = y;
+function TwoBucket(bucketOne, bucketTwo, goal, startBucket) {
+  this.bucketOne = bucketOne;
```

Comment:
I could agree or not about the new names, but I think giving better names than `x`, `y` or `z` would always help, so thank you so much for the refactor.